RecoLocalCalo/HcalRecAlgos/src/HBHETimeProfileStatusBitSetter.cc

```diff
@@ -47,19 +47,21 @@ HBHETimeProfileStatusBitSetter::HBHETimeProfileStatusBitSetter(double R1Min, dou
 HBHETimeProfileStatusBitSetter::~HBHETimeProfileStatusBitSetter(){}
 
 
+bool compareDigiEnergy(const HBHEDataFrame& x, const HBHEDataFrame& y)
+{
+  double TotalX=0, TotalY=0;
```

Comment:
this edit is a good chance to conform to the cmssw naming style: lower case for variable names